python/cac_tripplanner/templates/manifest.json

```diff
@@ -0,0 +1,79 @@
+{
+  "short_name": "GoPhillyGo",
+  "name": "GoPhillyGo Trip Planner",
```

Comment:
Here's another instance of "Trip Planner" to be removed. This is what ends up in the PWA splash.